src/styles/blocks/stopwatch-container.scss

```diff
@@ -0,0 +1,5 @@
+.stopwatch-container {
```

Comment:
this file could be removed because [stopwatch.scss](https://github.com/mate-academy/layout_stop-watch/pull/2118/files#diff-d40d9de6b8521ee073b31965231fe8ed17fdd4c525e5760f995f0821833c91d1) has the same styles